### PR TITLE
Refactor getting started

### PR DESCRIPTION
**Pull request by dokshina**
* Example directory contains all files for running the role
  (`Vagrantfile`, `playbook.yml`, `hosts.yml`)
* File `hosts.updated.yml` is a new inventory

### CODE REVIEW

File: examples/getting-started-app/hosts.updated.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+all:
+  vars:
+    cartridge_app_name: getting-started-app  # application name
+    cartridge_package_path: ./getting-started-app-1.0.0-0.rpm  # path to package to deploy
+
+    cartridge_cluster_cookie: app-default-cookie  # cluster cookie
+    cartridge_defaults:  # default instance parameters
```
Comment (olegrok) [inline examples/getting-started-app/hosts.updated.yml:8]: maybe "default tarantool instance parameters"?

File: examples/getting-started-app/README.md [hunk 2]
```diff
@@ -84,139 +92,14 @@ You also need to install `vagrant-hosts`, `vagrant-vbguest` and `vagrant-scp` pl
...
-
-To use the API, you can connect to the instance console using a socket:
+## Deploying with Ansible
 
-```bash
-$ tarantoolctl connect /var/run/tarantool/getting-started-app.storage-1.control
-tarantool> require('cartridge').is_healthy()
-```
+Now, we will start 3 instances on 2 different servers, join them in 2 replica
+sets, bootstrap vshard, enable automatic failover, set up authorization and
```
Comment (olegrok) [inline examples/getting-started-app/README.md:98]: > failover

I think it should be noticed that current implementation supports only "eventual" failover mode. At least two words in "Manage failover" section

File: examples/getting-started-app/README.md [hunk 3]
```diff
@@ -417,7 +300,7 @@ Tarantool be automatically installed with your application.
 If you want to install Tarantool yourself (e.g. from a package), you can set
 the `cartridge_enable_tarantool_repo` variable to `false`.
 
-#### Check if instances was started
+#### Check instances systemd services
```
Comment (olegrok) [inline examples/getting-started-app/README.md]: Seems "instances" is redundant word

File: examples/getting-started-app/README.md (hunk 2 above)
Comment (dokshina) [reply examples/getting-started-app/README.md:98]: Added a note to this README and the main README.

File: examples/getting-started-app/README.md [hunk 4]
```diff
@@ -35,6 +31,31 @@ for *developing* a Tarantool Cartridge application.
 Now you will learn how to *deploy* this application on servers and
 *set up the topology* using Ansible.
 
+This example directory contains `getting-started-app-1.0.0-0.rpm` file.
+It's the RPM package with the example application, we will deploy in on two
+virtual machines described in the `Vagrantfile`.
```
Comment (olegrok) [inline examples/getting-started-app/README.md]: You can put `Vagrantfile ` as a link to it.

**Review by olegrok — APPROVED** [review]

**Pull request merged**